src/services/svc-service.ts

```diff
@@ -75,6 +82,50 @@ export class SvcService {
     })
   }
 
+  public startServices(services: string[]): void {
+    const activeServices = this.getActiveServices()
+
+    if (services.length === 0) {
+      throw new CLIError(
+        'There are no enabled services. Please activate a service using the command: <%= config.bin %> enable <your_service_yml>'
+      )
+    }
+
+    const serviceNotFound = services.filter(
+      service => !activeServices.includes(service)
+    )
+    if (serviceNotFound.length > 0) {
+      const addS = serviceNotFound.length > 1
+      throw new CLIError(
+        `Service${addS ? 's' : ''} ${serviceNotFound.join(', ')} ${
+          addS ? 'are' : 'is'
+        } not enabled. Please check the enabled services with command: <%= config.bin %> list`
+      )
+    }
+
+    SvcService.debug(`starting service ${services.join(', ')}`)
+
+    try {
+      cp.execSync(
```

Comment:
I suggest using `ProcessExecutorService`. Refer to @zonia3000 or @alepintus if you have doubts